src/Utilities/TestingUtilities.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import configureStore from 'redux-mock-store';
+import { Provider } from 'react-redux';
+import { MemoryRouter, Routes, Route } from 'react-router-dom';
+import { IntlProvider } from 'react-intl';
+
+export const ComponentWithContext = ({
+    Component,
```

Comment:
Why not instead of `Component` and `componentProps` use `children` so the tests can invoke the component like any other:


```js
 <ComponentWithContext renderOptions={renderOptions}>
   <ImmutableDevices />
 </ComponentWithContext>
```